src/Security/Authentication/OpenIdConnect/samples/OpenIdConnect.AzureAdSample/Startup.cs

```diff
@@ -144,7 +144,7 @@ public void Configure(IApplicationBuilder app)
                             string userObjectID = context.User.FindFirst("http://schemas.microsoft.com/identity/claims/objectidentifier").Value;
                             var result = await authContext.AcquireTokenSilentAsync(Resource, credential, new UserIdentifier(userObjectID, UserIdentifierType.UniqueId));
 
-                            await response.WriteAsync($"<h3>access_token</h3><code>{HtmlEncode(result.AccessToken)}</code><br>");
+                            await response.WriteAsync($"<h3>access_token</h3><c>{HtmlEncode(result.AccessToken)}</c><br>");
```

Comment:
Another mis-target. This is HTML content in a sample and should not be changed to `<c>`.